src/raudio.c

```diff
@@ -476,6 +494,11 @@ void InitAudioDevice(void)
     TRACELOG(LOG_INFO, "    > Periods size:  %d", AUDIO.System.device.playback.internalPeriodSizeInFrames*AUDIO.System.device.playback.internalPeriods);
 
     AUDIO.System.isReady = true;
+
+    deviceInfo.sampleRate = AUDIO.System.device.playback.internalSampleRate;
```

Comment:
No need to store this info, it can be just queried by the users requiring it with a `GetAudioDeviceInfo()`